### PR TITLE
Prod Deploy 12-23-20

### PR DESCRIPTION
**Pull request by silasbaur**

### CODE REVIEW

File: _layouts/onsite-groups/location.html [hunk 1]
```diff
@@ -19,15 +19,15 @@ <h1 class="font-family-condensed-extra text-uppercase">{{ page.location.title }}
           {% endfor %}
         </div>
         <div>
-          <a class="text-white underline" href="/groups/onsite">See other group locations</a>
+          <a class="text-white underline" href="/groups/site-based#locations">See other group locations</a>
```
Comment (dholloran-ample) [inline _layouts/onsite-groups/location.html:22]: This doesn't seem to link to anything. Was the intention to link down the page to locations or something else?

**Review by tcmacdonald — APPROVED** [review]

File: redirects.csv [hunk 2]
```diff
@@ -53,7 +53,6 @@ https://discipleship.crossroads.net/*,https://www.crossroads.net/huddle,301!,mas
 /profile/personal,/profile/personal-info,302!
 /profile/*,https://${env:CRDS_PROFILE_ENDPOINT}/:splat,200! Role=user
 /profile/*,/signin?redirectUrl=/profile/:splat,302!
-/signin,https://${env:CRDS_ANGULAR_ENDPOINT}/signin,200!
```
Comment (dholloran-ample) [inline redirects.csv:56]: Should this go to prod I think Devy may be working on testing Okta ([Slack Message](https://crdschurch.slack.com/archives/C02C3F925/p1608231715112100)).

**Review by dholloran-ample — CHANGES_REQUESTED** [review]
@silasbaur - I left 2 comments/questions

File: redirects.csv (hunk 2 above)
Comment (silasbaur) [reply redirects.csv:56]: This should not go into prod

File: _layouts/onsite-groups/location.html (hunk 1 above)
Comment (silasbaur) [reply _layouts/onsite-groups/location.html:22]: I can fix that

**Review by dholloran-ample — APPROVED** [review]

**Pull request merged**